### ISSUE TITLE
syntax highlighting

### ISSUE THREAD

**Issue by zmoon** [Beliavsky/FortranTip#24]
It looks nice with carbon, but also exposing some issues with Highlight.js Fortran support, like in this one:
https://twitter.com/fortrantip/status/1477632693617799169/photo/1

**Comment by Beliavsky** [comment]
Yes, I noticed that. Trouble with "word", len_trim etc. I may post a request for help on Fortran Discourse.

**Comment by zmoon** [comment]
Also I feel like `intent` and `character` should get the same color, but maybe I am wrong?

**Comment by zmoon** [comment]
Also in that theme the two red-ish colors are pretty hard to tell apart, at least on my laptop screen

**Comment by Beliavsky** [comment]
I raised an issue: https://github.com/highlightjs/highlight.js/issues/3450 .

**Comment by joshgoebel** [comment]
Suggest filing an issue/support request with Carbon.

**Comment by zmoon** [comment]
https://github.com/carbon-app/carbon/issues/1308 is now closed, the issue in the example with `words` and such is fixed.

![image](https://user-images.githubusercontent.com/15079414/151032989-025e6dec-d94b-480d-9d05-0125d2122e40.png)
